R-proj/src/ess.cpp

```diff
@@ -0,0 +1,44 @@
+// [[Rcpp::depends(BH)]]
```

Comment:
It seems that the pattern is the same with existing project files, including previous MCMC diagnostics, so that seemed to be a design choice that existed a priori. Would it be better to merge to one file? 

Also mentioning @TolisChal

Reply:
I think it's better to have a new PR for that since it affects the R interface.

Reply:
I agree with tolis. It will pollute the current PR. I am resolving it.